src/DotNetty.Buffers/AbstractByteBuffer.cs

```diff
@@ -302,6 +303,12 @@ public virtual long GetLong(int index)
 
         public virtual char GetChar(int index) => Convert.ToChar(this.GetShort(index));
 
+        public virtual float GetFloat(int index)
+        {
+            int raw = this.GetInt(index);
+            return Unsafe.As<int, float>(ref raw);
```

Comment:
@yyjdelete we also do quite a bit of unsafe here:
https://github.com/Azure/DotNetty/blob/dev/src/DotNetty.Codecs/Base64/Base64.cs#L26